clients/benchmarks/client.cpp

```diff
@@ -482,12 +482,12 @@ try
          "Apply bias vector")
 
         ("scaleA",
-         value<std::string>(&scaleAFormat)->default_value(""),
-         "Apply scale for A buffer. s = scalar, v = vector.")
+         value<int>(&scaleAFormat)->default_value(0),
```

Comment:
Using numbers instead of string is less straight forward. What are the pros and cons of using number instead of string?

Reply:
hipblaslt-test was already using the integers for scaleA and scaleB here:https://github.com/ROCm/hipBLASLt/blob/11dec36e1a1da2abfa8aefe65ebadfd8165a0dc1/clients/gtest/matmul_gtest.yaml#L571
So standardized it across both test and bench frameworks.